src/styles/blocks/header.scss

```
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: #00acdc, $transition-time;
```

```suggestion
    transition: color $transition-time;
```
check syntax, now it doesn't work